redis-operator.yaml

```
@@ -0,0 +1,45 @@
+package:
+  name: redis-operator
+  version: "0.18.1"
+  epoch: 1
```

```suggestion
  epoch: 0
```
we start with epoch o and bump it on every rebuild.